user/monolith/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "monolith"
+pkgver = "2.8.3"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["pkgconf"]
+makedepends = [
+    "rust-std",
+    "cargo-auditable",
+    "openssl-devel",
+]
+
+env = {
+    "OPENSSL_NO_VENDOR": "1",
```

Comment:
this is already a default for cargo build_style consumers https://github.com/chimera-linux/cports/blob/d134077/src/cbuild/util/cargo.py#L35-L36, please don't copy other distro packaging vebatem, they should be references for when things don't build and you can't figure something out quickly by yourself

also for future avoid adding unnecessary newlines around default template variables like `env` here since it breaks checking for correct template variable order